packages/manager/apps/pci-kubernetes/src/helpers/index.ts

```diff
@@ -1,4 +1,7 @@
 import { PaginationState } from '@ovh-ux/manager-react-components';
+import { Query } from '@tanstack/react-query';
+
+export const REFETCH_INTERVAL_DURATION = 15000;
```

Comment:
```suggestion
export const REFETCH_INTERVAL_DURATION = 15_000;
```

Not important but useful for the readability ;)